Fixes tabs, removes broken link, and other minor style changes. (#1152)
saucelabs/sauce-docs: docs/basics/integrations/deque.md

## Sauce Labs UI

Sauce Labs will display the accessibility results in our UI as a tab in the job itself:

<img src={useBaseUrl('img/accessibility/example.png')} alt="Accessibility Example in Sauce UI"/>

## Language Specific Examples

<Tabs
  groupId="axe-lang-examples"
  defaultValue="java"
  values={[
    {label: 'Java', value: 'java'},

>     {label: 'WebdriverIO', value: 'webdriver'},

Reviewer: ```suggestion
    {label: 'JavaScript', value: 'js'},
```

Reviewer: Either way `webdriver` is confusing because WebdriverIO is not abbreviated as WebDriver. Better to keep them all as languages.

    {label: 'Python', value: 'python'},
    {label: 'Ruby', value: 'ruby'},

>     {label: 'C#', value: 'c#'},

Reviewer: agreed we shouldn't be using dotnet (which is how Selenium refers to its bindings for some reason)
We should use either `cs` or `csharp` to keep it inline with the highlight tags https://rdmd.readme.io/docs/code-blocks#language-support

Reviewer 2: that is good to know!

Author: Changed to `csharp`

>     {label: 'C#', value: 'c#'},

Reviewer: ```suggestion
    {label: 'C#', value: 'csharp'},
```

__Example Code__

> ```python reference title="Accessibility Test with sa11y"

Reviewer: capitalized is correct. It is the Proper noun name of the project.

> ```python reference title="Accessibility Test with sa11y"

Reviewer: ```suggestion
```python reference title="Accessibility Test with Sa11y"
```

__Requirements__

1. Add `sa11y` to your `gemfile`:
  ```shell
  gem 'sa11y', '~> 0.2.1'
  ```

> 1. Require the `analyze` class in your file:

Reviewer: Ruby classes are capitalized

> 1. Require the `analyze` class in your file:

Reviewer: ```suggestion
1. Require the `Analyze` class in your file:
```

__Example Code__

```ruby reference title="Accessibility Test with Sa11y"
https://github.com/saucelabs-training/demo-ruby/blob/docs-1.0/sauce-features/accessibility/spec/sa11y_spec.rb
```

</TabItem>
<TabItem value="c#">

> To get accessibility results with C# and .NET, we recommend [Selenium.Axe for .NET](https://github.com/TroyWalshProf/SeleniumAxeDotnet/). Make sure you are using version 3.x.

Reviewer 2: ```suggestion
To get accessibility results with C# and .NET, we recommend using [Selenium.Axe for .NET](https://github.com/TroyWalshProf/SeleniumAxeDotnet/), version 3.0 or higher.
```
